package.json

```diff
@@ -38,12 +38,12 @@
   "scripts": {},
   "repository": {
     "type": "git",
-    "url": "https://github.com/theirc/Service-Mapper.git"
+    "url": "https://github.com/theirc/ServiceInfo.git"
```

Comment:
Should these be `ServiceInfo-project`? I guess they don't really matter because it won't be pushed to NPM.